src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -34,18 +34,19 @@ public function process(ContainerBuilder $container)
         $types = array();
 
         foreach ($container->findTaggedServiceIds('form.type') as $serviceId => $tag) {
+            // Support type access by FQCN
+            $serviceDefinition = $container->findDefinition($serviceId);
```

Comment:
You can keep using ``getDefinition`` here. No need to use ``findDefinition`` to resolve aliases, as aliases cannot be tagged, and you are not altering the container in the loop.